documentation/Using-the-SDK/Creating-your-plugin.md

```diff
@@ -10,46 +10,24 @@ This document will cover:
 
 
 
-For simplicity, this section will assume you are using Visual Studio 2019. The instructions may be adapted for other editors / IDEs.
+For simplicity, this section will assume you are using Visual Studio. The instructions may be adapted for other editors / IDEs.
 
 <a name="reqs"></a>
 # Requirements
 
-1. [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+1. [Visual Studio](https://visualstudio.microsoft.com/downloads/)
 2. [.NET SDK that supports .NET Standard 2.0](https://dotnet.microsoft.com/download/visual-studio-sdks)
    * [See .NET Standard 2.0 support options](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
 
-## Visual Studio 2019
```

Comment:
I didn't see the benefit of having this here. It adds clutter to our already-complicated docs